cypress-shared/integration/affirm/2.1-graphql.spec.js

```diff
@@ -0,0 +1,35 @@
+import { updateRetry, loginViaCookies } from '../../support/common/support'
+import {
+  graphql,
+  version,
+  affirmSettings,
+  orderData,
+  orderUpdate,
+  validateGetVersionResponse,
+  validateAffirmSettingsResponse,
+  validateOrderDataResponse,
+  validateOrderUpdateResponse,
+} from '../../support/affirm-payment/graphql_testcase'
+import { orderInfo } from '../../support/affirm-payment/outputvalidation'
+
+const prefix = 'Graphql testcase'
+
+describe('Affirm GraphQL Validation', () => {
+  loginViaCookies({ storeFrontCookie: true })
```

Comment:
You don't need to pass storeFrontCookie true 
Because By default it is true

Reply:
@Syed-Vtex  removed the storeFrontCookie: true